app/controllers/responsible_body/devices/schools_controller.rb

```diff
@@ -1,7 +1,9 @@
 class ResponsibleBody::Devices::SchoolsController < ResponsibleBody::BaseController
   before_action :load_schools_by_order_status, only: %i[show index]
 
-  def index; end
+  def index
+    @show_devices_ordered_column = @responsible_body.virtual_cap_pools.none?
```

Comment:
Might also need to check `@responsible_body.has_virtual_cap_feature_flags?`, just in case? (Not sure if it really matters, but I can imagine a situation with the data where the RB has vcap pools but then the flag for that particular rb was turned off for some reason).

Reply:
I'll add that check too to be safe.